main.go

```diff
@@ -8,11 +8,11 @@ import (
 
 // GitBranch is set by the CI build process to the name of the branch
 //nolint:gochecknoglobals // This is filled in by the build system
-var GitBranch string = "local"
+var GitBranch = "local"
 
 // GitCommit is set by the CI build process to the commit hash
 //nolint:gochecknoglobals // This is filled in by the build system
-var GitCommit string = "build"
+var GitCommit = "build"
```

Comment:
also here, why did you changed it? and if so, lets make just `GitBranch := "local"`

Reply:
I think just because the `string` type declaration was unnecessary.

you can't use `:=` where these variables are being declared.

Reply:
> you can't use := where these variables are being declared.

oh, ok, I forgot, sorry